.github/workflows/main.yml

```
@@ -130,5 +130,44 @@ jobs:
         with:
          name: app reports
          path: app/build/test-results
-      
+
+  bazel_build_app:
+    name: Build Binary with Bazel
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-18.04]
+    steps:
+      - uses: actions/checkout@v2
+      - name: Setup bazel
```

```suggestion
      - name: Set up bazel
```

(set up is a verb vs noun setup)